src/components/useRealtimeAPI.tsx

```
@@ -0,0 +1,263 @@
+              state: jsonData.type,
+            })
+          }
+          break
+        case 'conversation.item.input_audio_transcription.completed':
+          console.log('音声データの音声認識が完了しました', jsonData)
+          break
+      }
+
+      if (
+        (jsonData.type === 'response.audio.delta' &&
+          accumulatedAudioDataRef.current?.buffer.byteLength > 100_000) ||
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)